internal/pkg/agent/cmd/validate.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-agent/internal/pkg/agent/application/paths"
+	"github.com/elastic/elastic-agent/internal/pkg/cli"
+	"github.com/elastic/elastic-agent/internal/pkg/otel"
+)
+
+func newValidateCommandWithArgs(_ []string, _ *cli.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "validate",
+		Short:         "Validates the OpenTelemetry collector configuration without running the collector",
+		SilenceUsage:  true, // do not display usage on error
+		SilenceErrors: true,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := cmd.Context()
+			cfgFile := paths.ConfigFileWithDefault("otel.yml")
+			return validateOtelConfig(ctx, cfgFile)
+		},
+	}
+
+	return cmd
+}
+
+func validateOtelConfig(ctx context.Context, cfgFile string) error {
+	if runAsOtel := otel.IsOtelConfig(ctx, cfgFile); !runAsOtel {
```

Comment:
Any reason this shouldn't be shortened?

```suggestion
	if !otel.IsOtelConfig(ctx, cfgFile) {
```